applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -82,9 +82,22 @@ static std::ostream& pythonToSofaDataString(PyObject* value, std::ostream& out)
         return out << PyString_AsString(value) ;
     }
 
+    /// Unicode are converted to string.
+    if(PyUnicode_Check(value))
+    {
+        PyObject* tmpstr = PyUnicode_AsUTF8String(value);
+        out << PyString_AsString(tmpstr) ;
+        Py_DECREF(tmpstr);
+
+        return out;
+    }
 
     if( PySequence_Check(value) )
     {
+        if(!PyList_Check(value))
```

Comment:
Hi Evan,

The adequate level of warning was discussed previously and if I remember correctly @maxime-tournier  pointed that having a warning each time we are passing something else than a list was a bit too restrictive.